CHANGELOG.md

```diff
@@ -16,6 +16,10 @@
 * Improve error and warning messages when targetting an xcdatamodeld.  
   [Steven Watremez](https://github.com/StevenWatremez)
   [#53](https://github.com/NijiDigital/gyro/pull/53)
+- Fix nullable enum methods in android-kotlin.
```

Comment:
Please add two spaces after the period, for proper markdown rendering

Reply:
✓